src/BuoyancyModels/buoyancy.jl

```diff
@@ -30,28 +30,36 @@ model = NonhydrostaticModel(grid=grid, buoyancy=buoyancy, tracers=:b)
 
 # output
 
+┌ Warning: The meaning of `gravity_unit_vector` changed in version 0.80.0.
+│ In versions 0.79 and earlier, `gravity_unit_vector` indicated the direction _opposite_ to gravity.
+│ In versions 0.80.0 and later, `gravity_unit_vector` indicates the direction of gravitational acceleration.
+└ @ Oceananigans.BuoyancyModels ~/builds/tartarus-16/clima/oceananigans/src/BuoyancyModels/buoyancy.jl:48
 NonhydrostaticModel{CPU, RectilinearGrid}(time = 0 seconds, iteration = 0)
 ├── grid: 1×8×8 RectilinearGrid{Float64, Periodic, Periodic, Bounded} on CPU with 3×3×3 halo
 ├── timestepper: QuasiAdamsBashforth2TimeStepper
 ├── tracers: b
 ├── closure: Nothing
-├── buoyancy: BuoyancyTracer with -ĝ = Tuple{Int64, Float64, Float64}
+├── buoyancy: BuoyancyTracer with ĝ = Tuple{Int64, Float64, Float64}
 └── coriolis: Nothing
 ```
 """
-function Buoyancy(; model, gravity_unit_vector=ZDirection())
+function Buoyancy(; model, gravity_unit_vector=NegativeZDirection())
+    gravity_unit_vector != NegativeZDirection() &&
+        @warn """The meaning of `gravity_unit_vector` changed in version 0.80.0.
+                 In versions 0.79 and earlier, `gravity_unit_vector` indicated the direction _opposite_ to gravity.
+                 In versions 0.80.0 and later, `gravity_unit_vector` indicates the direction of gravitational acceleration."""
```

Comment:
```suggestion
        @warn """The `gravity_unit_vector` meaning changed in version 0.80.0.
                 Up to version 0.79, `gravity_unit_vector` indicates the direction _opposite_ to gravity.
                 From version 0.80.0 and later, `gravity_unit_vector` indicates the direction of gravitational acceleration."""
```

Reply:
I had something that was close to what you wrote, but @glwagner suggested the current version and I accepted it. I do think the current version is clearer (albeit more repetitive).

Reply:
Let's leave it then! :)